### PR TITLE
Branch for release 1.2.0

### PR DESCRIPTION
**Pull request by prusse-martin**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,14 @@
 History
 =======
 
+1.2.0 (2019-05-07)
+------------------
+
+* Crude fix for issue #91 with pip dependences. (`#92_`).
+
+.. _`#72`: https://github.com/ESSS/conda-devenv/pull/92
```
Comment (nicoddemus) [inline HISTORY.rst]:
```suggestion
.. _`#92`: https://github.com/ESSS/conda-devenv/pull/92
```

File: HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,14 @@
 History
 =======
 
+1.2.0 (2019-05-07)
+------------------
+
+* Crude fix for issue #91 with pip dependences. (`#92_`).
```
Comment (nicoddemus) [inline HISTORY.rst]:
```suggestion
* Correctly support git and mercurial repositories in ``pip`` dependencies. (`#92_`).
```

**Comment by prusse-martin** [comment]
The version 1.1.2 including this fix has been released.
Thank you for this fix.
Soon this will be available on conda-forge.

**Comment by nicoddemus** [comment]
cc @mforbes

**Pull request merged**